ftdetect/markdown.vim

```diff
@@ -1,3 +1,3 @@
 " markdown filetype file
-au BufRead,BufNewFile *.{md,mdown,mkd,mkdn,markdown,mdwn}   set filetype=mkd.markdown
```

Comment:
@yuex why don't we keep `mkd.markdown` so as to not break user scripts, and add a README note explaining that `mkd` is deprecated?